Automattic/wp-calypso
# Move reader-popover into reader/components

### Changes proposed in this Pull Request
A number of Reader-specific components and blocks currently live in the main components and blocks directories, even though they are not used elsewhere in Calypso. These components and blocks should live inside client/reader if they are not likely to be reused outside Reader. See also: #28161 and #28272.

### What's reader-popover anyway?
> This is a wrapper around Popover used in Reader. It behaves identically to Popover, but has a slightly different look and feel and adds an optional title. To avoid style duplication, it's wrapped up in this component.

### Testing instructions
Load the Reader at http://calypso.localhost:3000.

Check that the share popover looks like this:

![screen shot 2018-11-06 at 15 35 39](https://user-images.githubusercontent.com/17325/48039667-bd4f2180-e1da-11e8-9f23-d83f1ff15748.png)

In Following Manage at http://calypso.localhost:3000/following/manage, check that the Notification Settings popover looks like this:

![screen shot 2018-11-06 at 15 36 12](https://user-images.githubusercontent.com/17325/48039694-d788ff80-e1da-11e8-99aa-c3d80cc99e70.png)
